docker/image.ts

```diff
@@ -120,6 +125,7 @@ export class Image extends pulumi.ComponentResource {
             baseImageName: this.baseImageName,
             imageName: this.imageName,
             digest: this.digest,
+            registryServer: imageData.apply(d => d.registryServer),
```

Comment:
I think we also need to assign `this.registryServer` somewhere?

Reply:
good catch.  thanks.